panel/src/components/Lab/PlaygroundView.vue

```diff
@@ -106,7 +103,7 @@ export default {
 </script>
 
 <style>
-.k-lab-playground-view[data-has-tabs="true"] .k-lab-playground-header {
+.k-lab-playground-view:has(.k-tabs) .k-lab-playground-header {
```

Comment:
I would say that this is more risky because it could get triggered by an unexpected .k-tabs component somewhere (i.e. in a nested component). I would keep the data attribute for more control.

Reply:
How about `:has(> .k-tabs)`?